pallets/orml-currencies-allowance-extension/src/default_weights.rs

```diff
@@ -0,0 +1,67 @@
+
+//! Autogenerated weights for orml_currencies_allowance_extension
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2023-03-20, STEPS: `100`, REPEAT: 10, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! HOSTNAME: `MacBook-Pro`, CPU: `<UNKNOWN>`
+//! EXECUTION: None, WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
+
+// Executed Command:
+// ./target/release/spacewalk-standalone
+// benchmark
+// pallet
+// --chain=dev
+// --pallet=orml-currencies-allowance-extension
+// --extrinsic=*
+// --steps=100
+// --repeat=10
+// --wasm-execution=compiled
+// --output=pallets/default_weights.rs
+// --template=./.maintain/frame-weight-template.hbs
```

Comment:
How did you actually run the benchmarks? The comment says it was executed on the spacewalk-standalone chain? 🤔

Reply:
Yes. I added the pallet to standalone spacewalk node to generate a default weights.

Reply:
Hmm okay, I see. Setting up benchmarking for our pendulum chains should probably happen in another PR with another issue just for that. I'll create that ticket and your changes these changes are fine for now then. We should consider re-running the benchmarks on our pendulum chains once it's set up properly though.